<?xml version="1.0"?>
<launch>
    <!-- arm motion service -->
    <node pkg="arm_motion_service" type="arm_motion_server.py" name="arm_motion_service_left_arm" required="true" output="screen">
        <param name="arm_group_name" value="left_arm"/>
    </node>
</launch>
